doc/quick-start.rst

```diff
@@ -91,11 +110,19 @@ which you can dive deeper into Dune's capabilities:
   ``dune-project`` file whenever you build your project. You shouldn't need to
   worry about this, but you can see :ref:`opam-generation` for details.
 * The ``dune`` files in each directory specify the component to be built with
-  the files in that directory. For details on dune files, see :ref:`dune-files`.
+  the files in that directory. For details on ``dune`` files, see :ref:`dune-files`.
 
 Initializing a Library
 ----------------------
 
+There are two kinds of libraries: a shared library and a static library. A shared 
```

Comment:
The descriptions here seem a bit muddled. This is also the first time I have heard of shared vs static libraries.

Reply:
I removed it.